hackathon_site/dashboard/frontend/src/components/teamDetail/SimpleOrderTables/SimpleOrderTables.tsx

```diff
@@ -0,0 +1,233 @@
+import { mockPendingOrdersInTable, mockReturnedOrdersInTable } from "testing/mockData";
+import React, { useState } from "react";
+import Container from "@material-ui/core/Container";
+import styles from "components/general/OrderTables/OrderTables.module.scss";
+import { Checkbox, FormGroup, Grid, TableCell, Tooltip } from "@material-ui/core";
+import Button from "@material-ui/core/Button";
+import {
+    GeneralOrderTitle,
+    GeneralPendingTable,
+    GeneralReturnTable,
+} from "components/general/OrderTables/OrderTables";
+import {
+    Field,
+    FieldProps,
+    Form,
+    Formik,
+    FormikValues,
+    useFormikContext,
+} from "formik";
+import FormControlLabel from "@material-ui/core/FormControlLabel";
+import { OrderInTable } from "api/types";
+
+interface SimpleOrderFormValues {
+    itemIdsChecked: string[];
+    checkAll: boolean;
+}
+
+const TableCheckbox = ({
+    field,
+    option,
+    order,
+    ...props
+}: FieldProps & { option: string; order: OrderInTable }) => {
+    const { values: formikValues, setFieldValue } =
+        useFormikContext<SimpleOrderFormValues>();
+
+    const recheckAllItems = (checked: boolean) => {
```

Comment:
This was actually such a smart way of doing this, I was confused for like 15 minutes till I realised what the purpose was LOL

Reply:
ty ty <3 this was very sweet